pkg/ceph/osd/health_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2017 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package osd
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/util/proc"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOSDStatus(t *testing.T) {
```

Comment:
this unit test could definitely use a few comments along the way about what your expectations are for each block of asserts and why certain functions have been mocked.  for the latter, maybe something as simple as this would suffice:
```
// we are mocking the OSD create path as well in this test so that we can mock the creation of an OSD process to stop once this test detects that it is down
```